docs/source/gbd2017_models/causes/neoplasms/breast_cancer_with_stage0/cancer_model.rst

```diff
@@ -0,0 +1,304 @@
+  +---------------+----------------------+-------+---------------------------------+----------------------------+
+  | All NCDs      | c_409                | 1     |                                 |                            |
+  +---------------+----------------------+-------+---------------------------------+----------------------------+
+  | Neoplasms     | C_410                | 2     |                                 |                            |
+  +---------------+----------------------+-------+---------------------------------+----------------------------+
+  | Breast cancer | C_429                | 3     | diagnosis_and_primary_therapy_phase_of_breast_cancer (s_277) |
+  |               |                      |       | metastatic_phase_of_breast_cancer (s_279)                    |
+  |               |                      |       | terminal_phase_of_breast_cancer (s_280)                      |
+  |               |                      |       | controlled_phase_of_breast_cancer_with_mastectomy (s_5486)   |
+  |               |                      |       | controlled_phase_of_breast_cancer_without_mastectomy (s_5489)| 
+  |               |                      |       | mastectomy_from_breast_cancer_beyond_ten_years (s_5492)      |
+  +---------------+----------------------+-------+--------------------------------------------------------------+
+
+.. image:: breast_cancer_hierarchy.svg
+
+
+Restrictions
+++++++++++++
+
+The following table describes any restrictions on the effects of this cause
+(such as being only fatal or only nonfatal), as well as restrictions on the age
+and sex of simulants to which different aspects of the cause model apply.
+
+.. list-table:: Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - YLL only
+     - False
+     -
+   * - YLD only
+     - False
+     -
+   * - YLL age group start
+     - 15 to 19
+     - GBD age group id 8
+   * - YLL age group end
+     - 95 plus
+     - GBD age group id 235
+   * - YLD age group start
+     - 15 to 19
+     - GBD age group id 8
+   * - YLD age group end
+     - 95 plus
+     - GBD age group id 235
+
+
+Vivarium Modeling Strategy
+--------------------------
+
+
+Scope
++++++
+
+.. todo::
+
+   Add scope.
+
+Model Assumptions and Limitations
++++++++++++++++++++++++++++++++++
+
+1. Within GBD 2017, after diagnosis/ treatment if a patient survives more than 10 years, they are considered cured for calculating disability. 
+For simulation models, this means that if the simulation is run for more than 10 years, then excess mortality rate exists due to cancer after 
+10 years and the number of deaths increase. But as per GBD 2017, after 10 years, the patients do not have excess mortality rate. So, this model 
+might over estimate deaths in that scenario.
+
```

Comment:
This is a little unclear to me. 

How does GBD model this explicitly? After 10 years are individuals removed from breast cancer prevalence or are they just moved to a sequela with a disability weight of zero? 

I think that this would be the case even if the simulation isn't run for more than 10 years, right? Because when we initialize a simulant into the breast cancer state, it means that they experienced breast cancer incidence sometime in the past. We don't assign a specific value to how long they've had it, but presumably it will be closer to 10 years for some simulants and closer to 0 years for others.

Should we allow individuals to remit after 10 years in our model? Seems like this would be easy enough if we set a duration transition of 10 years and initialized the population at simulation start with a uniform distribution of disease duration between 0 and 10 years?

Reply:
Not sure...Kiran wrote this and I just left it in. Have to dig a bit more to understand.

Reply:
Yeah, it seems like this is worth some further investigation.